### ISSUE TITLE
Issue running yarn command `danger` with `ci` arguments in Bitrise UI

### ISSUE THREAD

**Issue by justinmakaila** [jonoirwinrsa/steps-yarn#3]
I have a step in my .yml file that looks like this:
```
    - yarn@0.0.4:
        inputs:
        - args: ci
        - command: danger
        is_always_run: true
        title: Danger
```
However, the output from my build on Bitrise is:
```
+------------------------------------------------------------------------------+

| (4) yarn@0.0.4                                                               |
+------------------------------------------------------------------------------+
| id: yarn                                                                     |
| version: 0.0.4                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: bash                                                                |
| time: 2018-02-07T07:21:13-08:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[07:21:13]  * [OK] Step dependency (node) installed, available. 
INFO[07:21:13]  * [OK] Step dependency (yarn) installed, available. 
==> Switching to working directory: /Users/vagrant/git
yarn run v1.3.2
$ /Users/vagrant/git/node_modules/.bin/danger
  Usage: danger [options] [command]
  Options:
    -V, --version  output the version number
    -h, --help     output usage information
  Commands:
    init        Helps you get started with Danger
    ci          Runs Danger on CI
    process     Like `ci` but lets another process handle evaluating a Dangerfile
    pr          Runs your local Dangerfile against an existing GitHub PR. Will not post on the PR
    runner      Runs a dangerfile against a DSL passed in via STDIN [You probably don't need this]
    local       Runs danger standalone on a repo, useful for git hooks
    help [cmd]  display help for [cmd]
  Docs:
    -> Getting started:
       http://danger.systems/js/guides/getting_started.html
    -> The Dangerfile
       http://danger.systems/js/guides/the_dangerfile.html
    -> API Reference
       http://danger.systems/js/reference.html
Done in 0.30s.
```

I suspect this is due to this: https://github.com/jonoirwinrsa/steps-yarn/blob/master/step.yml#L36,
where it looks like the arguments are passed to a variable `command` and `args`, but the script only references `command`: https://github.com/jonoirwinrsa/steps-yarn/blob/master/step.sh#L28

**Comment by jonoirwinrsa** [comment]
HI @justinmakaila 

Yeah I think that is a problem.. I don't know if there is too much point to using this step now - I believe that yarn comes pre-installed now so a cleaner solution might be to run this step as a script instead..

I will update this step regardless to include the args

**Comment by jonoirwinrsa** [comment]
Hi @justinmakaila sorry for being so slow about this but the issue should now be fixed in version 0.0.6 of the step which is waiting to be reviewed.

Closing this for now.
